src/puzzle-pieces/pm-puzzle-pieces.ts

```diff
@@ -358,6 +362,25 @@ customElements.define(
         );
         this.draggedPieceID = null;
       }
+      document.addEventListener("keydown", (event) => {
```

Comment:
A few issues with this one like it was mentioned in the Procedures doc.  I might temporarily disable this feature before pushing it live until those can be resolved.